src/math_functions.cpp

```diff
@@ -556,7 +556,7 @@ void calc_zn(int n, double rho, double phi, double zn[]) {
   // ===========================================================================
   // Calculate R_pq(rho)
   // Matrix forms of the coefficients which are easier to work with
-  std::vector<std::vector<double>> zn_mat(n + 1, std::vector<double>(n + 1));
+  vector<std::vector<double>> zn_mat(n + 1, std::vector<double>(n + 1));
```

Comment:
std:: in here

Reply:
Ah yeah. Forgot the trailing `g` in `sed`. Thanks!